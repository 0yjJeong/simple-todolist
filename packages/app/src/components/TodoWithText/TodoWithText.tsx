import styled from 'styled-components';

const TodoWithText = styled.div`
  display: flex;
  align-items: center;
  gap: 1rem;

  li {
    flex: 1;
  }
`;

export default TodoWithText;
