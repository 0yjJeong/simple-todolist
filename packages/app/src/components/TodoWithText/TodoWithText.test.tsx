import { render } from '@testing-library/react';
import 'jest-styled-components';
import TodoWithText from './TodoWithText';

describe('<ShadowBox/>', () => {
  it('works', () => {
    const rendered = render(<TodoWithText />);
    expect(rendered.container.firstChild).toMatchSnapshot();
    expect(rendered.container.firstChild).toHaveStyleRule('display', 'flex');
    expect(rendered.container.firstChild).toHaveStyleRule(
      'align-items',
      'center'
    );
  });
});
