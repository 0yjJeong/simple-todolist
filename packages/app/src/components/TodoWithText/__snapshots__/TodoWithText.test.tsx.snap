// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShadowBox/> works 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 1rem;
}

.c0 li {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

<div
  class="c0"
/>
`;
